docs/swagger.json

```diff
@@ -119,6 +119,78 @@
                 }
             }
         },
+        "/info/analysis/status/by/analysis-id/{analysis-id}": {
```

Comment:
It looks like this file is in `.gitignore` was it checked in by accident at some point?